### PR TITLE
Add a PR builder GitHub workflow.

### PR DESCRIPTION
**Pull request by HimashiRathnayake**
## Purpose
This PR adds a GitHub Action that builds the PR.

## Goals

- Adds GitHub Action that builds the PR.

### CODE REVIEW

File: .github/workflows/pr-builder.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - name: Bootstrap
+        id: install-npm-modules
+        run: npm run bootstrap
+      - name: Lint
+        id: lint
```
Comment (brionmario) [inline .github/workflows/pr-builder.yml]:
```suggestion
        id: lint-source-code
```

File: package.json [hunk 2]
```diff
@@ -31,7 +31,10 @@
         "start": "react-app-rewired start",
         "build": "react-app-rewired build",
         "test": "react-app-rewired test",
-        "eject": "react-scripts eject"
+        "eject": "react-scripts eject",
+        "bootstrap": "npm install",
```
Comment (brionmario) [inline package.json]: Lets make these alphabetically arranged.

File: .github/workflows/pr-builder.yml (hunk 1 above)
Comment (HimashiRathnayake) [reply .github/workflows/pr-builder.yml]: Fixed with [b4c5c67](https://github.com/wso2-incubator/identity-authentication-flow-composer/pull/13/commits/b4c5c67c7c60dbb6fb4f93ce3d8a62cb907a3d6a)

File: package.json (hunk 2 above)
Comment (HimashiRathnayake) [reply package.json]: Fixed with [39aac7f ](https://github.com/wso2-incubator/identity-authentication-flow-composer/pull/13/commits/39aac7f048076488d278c00d31462ed8481bdb25)

**Review by brionmario — APPROVED** [review]

**Pull request merged**